geoh5py/workspace/workspace.py

```diff
@@ -94,38 +95,34 @@ class Workspace(AbstractContextManager):
     }
 
     def __init__(
```

Comment:
**question:** now, should  the code always use `Workspace.create()` rather than instantiating directly with `Workspace()` ?

Reply:
well the straight instantiation would work if the h5file exists.

Reply:
**suggestion: (doc):** rename param to `existing_h5file` or have some doc for this param

Reply:
Ok but h5file right now can also be a BytesIO (technically a file but in memory). I will add docs.